Item/Item.lua

```diff
@@ -410,10 +410,34 @@ function methodtable.setCategories( self )
 		return
 	end
 
+	--- Lowers all words unless the word is fully capitalized (e.g.: EMP Generators -> EMP generators)
+	local function lowercaseWords( input )
+	    local result = input:gsub( '(%S+)' , function( word )
+    	    if word:match( ' ^[A-Z][A-Z]*$ ' ) then
+        	    return word
+        	else
+	            return string.lower( word )
+    	    end
+	    end)
+	    return result
+	end
+
 	--- Only set category if category_type value exists
 	if self.smwData[ translate( 'SMW_Type' ) ] ~= nil then
 		local typeCategory = translate( 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) )
-		if typeCategory ~= nil and typeCategory ~= 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) then table.insert( self.categories, typeCategory ) end
+		if typeCategory ~= nil and typeCategory ~= 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) then
+			table.insert( self.categories, typeCategory ) 
+			
+			if self.smwData[ translate( 'SMW_Size' ) ] ~= nil then
+				local sizeCategory = translate( 'SMW_Size' ) .. ' ' .. self.smwData[ translate( 'SMW_Size' ) ] .. ' ' .. lowercaseWords( translate( 'category_' .. string.lower( self.smwData[ translate( 'SMW_Type' ) ] ) ) )
```

Comment:
I think `lowercaseWords` is only applicable for categories on .tools.  
Where .tools uses for example `Size 1 generator`, .wiki would use `Größe 1 Generator`.

Reply:
if locale is `en`: use lowercaseWords?

Reply:
Thats how I would do it. You should be able to get the current language by calling `lang:getCode()`.